source/dub/internal/utils.d

```diff
@@ -40,8 +40,14 @@ NativePath getTempDir()
 NativePath getTempFile(string prefix, string extension = null)
 {
 	import std.uuid : randomUUID;
+    import std.array: replace;
```

Comment:
I think the existing file uses tabs whereas you use whitespace here. We should probably just convert all files to whitespace at some point...

Reply:
fixed